addons/servicegraph/README.md

```diff
@@ -37,6 +37,19 @@ All endpoints take these query parameters:
 - `filter_empty=true` will restrict the nodes and edges shown to only
   those that reflect non-zero traffic levels during the specified
   `time_horizon`. Deafult is `false`.
+  
+- `destination_namespace` will filter the nodes and edges show to only 
+  those where the destination workload is in the given namespace.
+  
+- `destination_workload` will filter the nodes and edges show to only 
```

Comment:
is this for showing only the direct calls for a given workload?  So, we would use this in combination with `source_workload` for showing the direct neighbors of a given workload ?

Reply:
Yes, this parameter will show the direct call for a given workload. But the combination of these parameters as AND relation, so when we have `source_worload` and `destination_workload` will see only the direct call between these two workloads